pkgs/servers/klipper/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, python2
+}: stdenv.mkDerivation rec {
+  name = "klipper";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "KevinOConnor";
+    repo = "klipper";
+    rev = "v${version}";
+    sha256 = "1ijy2ij9yii5hms10914i614wkjpsy0k4rbgnm6l594gphivdfm7";
+  } + "/klippy";
+
+  nativeBuildInputs = [ (python2.withPackages (p: with p; [ cffi pyserial greenlet jinja2 ])) ];
+
+  # mark the main entrypoint as an executable
+  postPatch = "chmod 755 ./klippy.py";
```

Comment:
Can this be added to `installPhase`? Or is it invoked during build?

Reply:
Moved

Reply:
Oh, actually, I need it there otherwise the shebang doesn't get fixed

Reply:
Fixed